python/pyspark/util.py

```diff
@@ -305,35 +307,22 @@ def inheritable_thread_target(f):
     >>> Thread(target=inheritable_thread_target(target_func)).start()  # doctest: +SKIP
     """
     from pyspark import SparkContext
-    if os.environ.get("PYSPARK_PIN_THREAD", "false").lower() == "true":
-        # Here's when the pinned-thread mode (PYSPARK_PIN_THREAD) is on.
-        sc = SparkContext._active_spark_context
 
-        # Get local properties from main thread
-        properties = sc._jsc.sc().getLocalProperties().clone()
+    if os.environ.get("PYSPARK_PIN_THREAD", "true").lower() == "true":
```

Comment:
This seems to be the change of default value when `PYSPARK_PIN_THREAD` is not set. Do we need to mention this in the PR description?

Reply:
Oh yeah, it was a mistake from #32429, and actually I meant it by:

> #32429 missed one place at inheritable_thread_target (https://github.com/apache/spark/blob/master/python/pyspark/util.py#L308).

Let me me more explicit on that